_posts/2020-07-21-why-paper-and-pencil-is-my-best-tool.markdown

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title:  "Why paper and pencil are my best tools"
+date:   2020-07-21 23:30:41 -0300
+categories: tools
+---
+Just try thinking about how much time we spend looking for an ideal tool to taking our notes, reminders, scheduling, sketching… A tool that give us freedom and flexibility to do everything we need. Yeah... so try looking around. Can you see this tool? There it is! A simple paper and pencil right next to you.
+
+I can’t imagine how many ideas and how much time I saved using this and I will try explain why this works so well to me.
+
+In all my programmer life I’ve tried to use many tools that could improve my capabilities to remember things, organize to do lists, improve creativity, etc. One thing that worked so well on all my needs giving me freedom, and combining all other tools that have separated and specialized functions, is paper and pencil.
```

Comment:
The socond frase sounds a little weird, there is nothing wrong, but doesn't look naturally in english: 
"One thing that worked so well on all my needs giving me freedom..."
What do you think about?
"The one that's meet all my needs and give me freedom..."

Reply:
Nice one. I apply some fixes in your suggestion too:

`The one that meets all my needs and gives me the freedom`